Cookbook/Style-guide/README.md

```diff
@@ -764,6 +762,8 @@ Only explicitly use `open`, `public`, and `internal` when you require a full acc
 
 Use access control as the leading property specifier. The only things that should come before access control are the `static` specifier or attributes such as `@IBAction`, `@IBOutlet` and `@discardableResult`.
 
+Use `private`, `fileprivate`, `internal`, and `public` visibility qualifiers on each function implemented in an extension, rather than have a single qualifier for the entire extension.
+
```

Comment:
It now applies to _all_ extensions, not just protocol's.

Reply:
@diegopetrucci The "Preferred / Not Preferred" examples that are currently after this paragraph are related to the access control order rule, so the point before the one you moved there.

I'd rather keep the existing "Preferred / Not Preferred" example about "access control as a leading property" near the related paragraph, and add a dedicated "Preferred / Not Preferred" snippet for the rule you moved here. Maybe moving that rule before the "access control as leading property" instead of after would also make more sense.

Reply:
updated